src/librustc_mir/borrow_check/mod.rs

```diff
@@ -423,7 +422,7 @@ fn downgrade_if_error(diag: &mut Diagnostic) {
 }
 
 pub struct MirBorrowckCtxt<'cx, 'tcx> {
-    infcx: &'cx InferCtxt<'cx, 'tcx>,
+    pub(crate) infcx: &'cx InferCtxt<'cx, 'tcx>,
```

Comment:
General nit in the PR: You can use the `crate` visibility instead.

Reply:
Yes, but I plan to move the borrowck code into its own crate, which won't have the feature gate enabled.

Reply:
Add the feature gate to the new crate?

Reply:
I though `crate` was used less in the borrow check code than it is, so I could avoid the feature flag.